examples/ts-react-chat/.env.example

```
@@ -2,6 +2,15 @@
 # Get yours at: https://platform.openai.com/api-keys
 OPENAI_API_KEY=
 
+# MySQL persistence demo (/mysql-persistence)
+# Use MYSQL_URL or the individual MYSQL_* settings below.
+# MYSQL_URL=mysql://root:password@127.0.0.1:3306/tanstack_ai_chat
+MYSQL_HOST=127.0.0.1
+MYSQL_PORT=3306
+MYSQL_USER=root
+MYSQL_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MYSQL_PASSWORD key should go before the MYSQL_PORT key

(UnorderedKey)